backend/sass/common.blocks/_setup.scss

```diff
@@ -195,11 +191,9 @@ $kadena-orange: #F3AB3C;
     line-height: initial;
 
     &.button_type_confirm {
-      width: 96%;
```

Comment:
The loss of this width makes the 'Continue' button display incorrectly on the following screen. Compared `develop` to this branch with restarts.